TRACONBoundaries.geojson

```diff
@@ -64,8 +64,8 @@
-    {"type":"Feature","properties":{"id":"CYQB","prefix":["CYQB"],"name":"Quebec Terminal"},"geometry":{"type":"MultiPolygon","coordinates":[[[[-72.133333333332999,46.766666666667],[-72.0,46.516666666667],[-71.5,46.2],[-71.408333333333005,46.208333333333002],[-71.302777777778005,46.384444444444],[-71.182695833333,46.397607777777999],[-71.08708,46.428089722221998],[-71.000467222221999,46.469486944444],[-70.925463888888999,46.520564722221998],[-70.864347222221994,46.579796388889001],[-70.818998333332999,46.645405833333001],[-70.79084,46.715418611110998],[-70.780791111111,46.787720555556],[-70.789228888888999,46.86012],[-70.815970277778007,46.930412777778002],[-70.860267777778006,46.996451111111],[-70.920824444443994,47.056208888889003],[-70.995827777778004,47.107846944443999],[-71.083,47.149770555556003],[-71.1796675,47.180681388888999],[-71.282845277777994,47.199620833333],[-71.389333333332999,47.206],[-71.495821388888999,47.199620833333],[-71.598999166666999,47.180681388888999],[-71.695666666666995,47.149770555556003],[-71.808333333332996,47.101388888888998],[-71.861111111111001,47.151666666666998],[-72.066388888888994,46.858333333333],[-72.133333333332999,46.766666666667]]]]}},
+    {"type":"Feature","properties":{"id":"CYOW","prefix":["CYOW","CPTL"],"name":"Ottawa Arrival/Departure"},"geometry":{"type":"MultiPolygon","coordinates":[[[[-74.922275277777999,45.085691388889003],[-74.966666666666995,45.066666666666997],[-74.95,44.980555555556002],[-75.036666666667003,44.945277777778003],[-75.5026325,44.86657],[-75.655,44.840833333333002],[-76.212877222222005,44.739624166666999],[-76.223856388889004,44.962676666667001],[-76.235555555556004,45.166666666666998],[-76.25,45.516666666667],[-76.258587222222005,45.681975555556001],[-76.266666666667007,45.8375],[-75.900504722221996,45.776000833333001],[-75.675833333333003,45.738888888889001],[-75.117222222221997,45.644444444443998],[-75.101520555555993,45.608086666666999],[-75.026148055556007,45.433569722222003],[-75.009013611111001,45.353140555556003],[-75.007993888889004,45.336870555555997],[-75.008372777778007,45.304304444444],[-75.011967222221998,45.271835833333],[-75.014963888889,45.255687777778],[-75.023334166666999,45.223661388888999],[-75.034834444444002,45.192124444443998],[-75.049403611111003,45.161229722221996],[-75.057816111110995,45.146069722222002],[-75.0596025,45.143285555555998],[-74.922275277777999,45.085691388889003]]]]}},
```

Comment:
I'm not sure this will work or not. @maiuswong can you confirm or not that listing the same prefix in multiple TRACONs will list both?